int32 id
string topColour
string bottomColour
float64 x
float64 y
